backend/handlers/edittrippsot.js

```diff
@@ -0,0 +1,27 @@
+const trip = require('../../database/triphelpers.js')
```

Comment:
This file is spelt incorrectly. Also please use camelCase or snake_case for file names.